tutordistro/distro/extra_commands/infrastructure/tutor_commands.py

```diff
@@ -69,17 +69,24 @@ def run_command(self, command: str):
             command (str): Tutor command.
         """
         try:
-            process = subprocess.run(
+            print(f'Running "{command}"')
```

Comment:
Is this print necessary? Why did we delete the previous one?

Reply:
It is not necessary, I just wanted to show in the logs that a command is running. And no special reason, I just saw the `print` from before saying something is running is more useful than the one showing the output

what do you think?

Reply:
If the other `print` didn't show anything useful, I agree to leave this one.